bin/inx-chronicle/config.template.toml

```diff
@@ -7,6 +7,9 @@ database_name = "chronicle"
 [api]
 port = 8042
 allow_origins = "0.0.0.0"
+# password = "password"
+password_hash = "c42cf2be3a442a29d8cd827a27099b0c"
+password_salt = "saltines"
```

Comment:
Shouldn't that be a hash too?

Reply:
No? We don't check the salt against an unhashed salt.